app/static/js/jquery.tmpl.js

```diff
@@ -0,0 +1,486 @@
+/*
+ * jQuery Templating Plugin
```

Comment:
I am in the process of moving the old framework to ember. jquery.tmpl was working for the old implementation, but can now be replaced with ember's templating engine once I figure out how to get the relevant JSON structure from the backend and subsequently render it.